core/controlplane/config/templates/cloud-config-controller

```diff
@@ -1001,6 +1001,20 @@ write_files:
              | base64 -d > $f
              mv -f $f ${encKey%.enc}
            done;
+
+           {{- if .AssetsConfig.HasAnyAuthTokens }}
+           authDir=/etc/kubernetes/auth
+           echo generating $authDir/computed-tokens.csv
+           echo > $authDir/computed-tokens.csv
+
+           {{- if .AssetsConfig.HasTLSBootstrapToken }}
+           echo $(cat $authDir/tls-bootstrap.token),kubelet-bootstrap,10001,system:kubelet-bootstrap >> $authDir/computed-tokens.csv
```

Comment:
Sorry for being late in commenting something like this but
* my preference after read through code is, the token file named `tls-bootstrap-token` rather than `tls-bootstrap.token` so that we can be consistent in the word `token` contained in basenames of both files `tls-bootstrap-token` and `tokens.csv`
  * It can be named like `tls-bootstra-token.txt` but I feels that it is too much 😃 and we already have various txt files without the `.txt` extentions anyway